### PR TITLE
Added some sensors to Haltech dash

### PR DESCRIPTION
**Pull request by zed65**

### CODE REVIEW

File: firmware/controllers/can/can_dash.cpp [hunk 1]
```diff
@@ -537,7 +537,7 @@ static void canDashboardBmwE90(CanCycle cycle) {
 void canDashboardHaltech(CanCycle cycle) {
 
 	uint16_t tmp;
-
+
```
Comment (rusefillc) [inline firmware/controllers/can/can_dash.cpp:540]: are we adding a couple of random spaces here?

File: firmware/controllers/can/can_dash.cpp [hunk 2]
```diff
@@ -914,14 +919,15 @@ void canDashboardHaltech(CanCycle cycle) {
...
+			uint8_t gear = Sensor::getOrZero(SensorType::DetectedGear);
 			msg[0] = 0x00;
 			msg[1] = 0x00;
 			msg[2] = 0x00;
 			msg[3] = 0x00;
 			msg[4] = 0x00;
 			msg[5] = 0x00;
 			msg[6] = 0x00;
-			msg[7] = 0x00;
+			msg[7] = (gear);
```
Comment (rusefillc) [inline firmware/controllers/can/can_dash.cpp:930]: (whybrackets)?

File: firmware/controllers/can/can_dash.cpp [hunk 3]
```diff
@@ -663,8 +663,12 @@ void canDashboardHaltech(CanCycle cycle) {
 			CanTxMessage msg(CanCategory::NBC, 0x471, 2);
 			msg[0] = 0x00;
 			msg[1] = 0x00;
-			msg[2] = 0x00;
-			msg[3] = 0x00;
+			/*AcceleratorPedal*/
+			tmp = (uint16_t)(Sensor::getOrZero(SensorType::AcceleratorPedal) * 10);
+			msg[2] = (tmp >> 8);
+			msg[3] = (tmp & 0x00ff);
```
Comment (rusefillc) [inline firmware/controllers/can/can_dash.cpp:669]: "tmp & 0x00ff" and "tmp >> 8" copy pasta seems to be demanding a helper method.

**Review by rusefillc — CHANGES_REQUESTED** [review]

**Comment by rusefillc** [comment]
@zed65 something strange happened here? Did you wipe out PR content?

**Comment by zed65** [comment]
Sorry, I don't know what happened. I'm not comfortable enough to do this github things.
And clearly I'm not a programmer..

**Comment by rusefillc** [comment]
@zed65 please review/test https://github.com/rusefi/rusefi/commit/6257957e10d46de1196469b5c09cea69a61f29fb?

@zed65 please consider a new attempt with the new API

**Pull request closed without merging**